packages/context/src/__tests__/unit/binding.unit.ts

```diff
@@ -172,6 +173,44 @@ describe('Binding', () => {
       expect(b.type).to.equal(BindingType.DYNAMIC_VALUE);
     });
 
+    it('support a factory to access context/binding/session', async () => {
+      const factory: ValueFactory<string> = ({
+        context,
+        binding: _binding,
+        options,
+      }) => {
+        return `Hello, ${context.name}#${
+          _binding.key
+        } ${options.session?.getBindingPath()}`;
+      };
+      const b = ctx.bind('msg').toDynamicValue(factory);
+      const value = await ctx.get<string>('msg');
+      expect(value).to.equal('Hello, test#msg msg');
+      expect(b.type).to.equal(BindingType.DYNAMIC_VALUE);
+    });
+
+    it('supports a factory to use context', () => {
```

Comment:
This test seems like it's pretty much the same as `it('support a factory to access context/binding/session')` above. Please rework it to make it more clear what are the important differences from the previous one, or remove it completely if it's not testing anything new.